packages/react-components/src/components/MeetingConferencePhoneInfo.tsx

```diff
@@ -0,0 +1,260 @@
+  /**
+   * Tool Phone Label
+   */
+  meetingConferencePhoneInfoModalToll?: string;
+  /**
+   * No phone number available message
+   */
+  meetingConferencePhoneInfoModalNoPhoneAvailable?: string;
+}
+
+/* @conditional-compile-remove(teams-meeting-conference) */
+/**
+ * @beta
+ * MeetingConferencePhoneInfoModal Component Props.
+ */
+export interface MeetingConferencePhoneInfoModalProps {
+  conferencePhoneInfoList: ConferencePhoneInfo[];
+  showModal?: boolean;
+  onDismissMeetingPhoneInfoSettings?: () => void;
+}
+
+/* @conditional-compile-remove(teams-meeting-conference) */
+/**
+ * @beta
+ * a component for setting spoken languages
+ */
+export const MeetingConferencePhoneInfoModal = (props: MeetingConferencePhoneInfoModalProps): JSX.Element => {
+  const { conferencePhoneInfoList, showModal, onDismissMeetingPhoneInfoSettings } = props;
+
+  const theme = useTheme();
+  const strings = useLocale().strings.MeetingConferencePhoneInfo;
+
+  const onDismiss = useCallback((): void => {
+    if (onDismissMeetingPhoneInfoSettings) {
+      onDismissMeetingPhoneInfoSettings();
+    }
+  }, [onDismissMeetingPhoneInfoSettings]);
+
+  const PhoneInfoModalStyle: Partial<IModalStyles> = useMemo(() => themedPhoneInfoModalStyle(theme), [theme]);
+
+  return (
+    <>
+      {
+        <Modal
+          titleAriaId={strings?.meetingConferencePhoneInfoModalTitle}
+          isOpen={showModal}
+          onDismiss={onDismiss}
+          isBlocking={true}
+          styles={PhoneInfoModalStyle}
+        >
+          <Stack horizontal horizontalAlign="space-between" verticalAlign="center" className={titleContainerClassName}>
+            <Text className={titleClassName}>{strings?.meetingConferencePhoneInfoModalTitle}</Text>
+            <IconButton
+              iconProps={{ iconName: 'Cancel' }}
+              ariaLabel={strings?.meetingConferencePhoneInfoModalTitle}
+              onClick={onDismiss}
+              style={{ color: theme.palette.black }}
+            />
+          </Stack>
+          {conferencePhoneInfoList.length === 0 && (
+            <Stack horizontal>
+              <Text className={stepTextStyle}>{strings?.meetingConferencePhoneInfoModalNoPhoneAvailable}</Text>
+            </Stack>
+          )}
+          {conferencePhoneInfoList.length > 0 && (
+            <Stack>
+              <Stack horizontal horizontalAlign="space-between" className={phoneInfoInctructionLine}>
+                <Stack.Item style={{ display: 'flex' }}>
+                  <Stack horizontal className={phoneInfoStep}>
+                    <Stack className={infoConnectionLinkStyle(theme)}></Stack>
+                    <Stack.Item className={phoneInfoIcon(theme)}>
+                      <Stack verticalAlign="center" horizontalAlign="center">
+                        <Icon
+                          iconName="PhoneNumberButton"
+                          style={{ color: theme.palette.themePrimary, padding: '8px' }}
```

Comment:
pull this style out and reuse, the same styling is used for all icons

Reply:
Moved to style and changed to rem